stdlib/tokenize.pyi

```diff
@@ -120,6 +125,8 @@ class Untokenizer:
     def add_whitespace(self, start: _Position) -> None: ...
     def untokenize(self, iterable: Iterable[_Token]) -> str: ...
     def compat(self, token: Sequence[int | str], iterable: Iterable[_Token]) -> None: ...
+    if sys.version_info >= (3, 13):
+        def escape_brackets(self, token: Iterable[str]) -> str: ...
```

Comment:
```suggestion
        def escape_brackets(self, token: str) -> str: ...
```
This technically works on any iterable of strs but it's clear it's meant to work on a single str.